gen/definitions/wireless_ssid.yaml

```diff
@@ -0,0 +1,524 @@
+name: Wireless SSID
+rest_endpoint: /networks/%v/wireless/ssids
+data_source_name_query: true
+doc_category: Wireless
```

Comment:
Shouldn't we enable `put_create` and `no_delete` here?

Reply:
put_create actually encodes a different behavior, where the the request would be sent to `/networks/%v/wireless/ssids` and we need to send it to `/networks/%v/wireless/ssids/%v`, so i encoded that manually. i added no_delete though